packages/bot-web-ui/src/pages/dashboard/announcements/config.tsx

```diff
@@ -1,22 +1,25 @@
 import React from 'react';
-import { localize } from '@deriv/translations';
+import { Localize, localize } from '@deriv/translations';
 import { DBOT_TABS } from 'Constants/bot-contents';
 import { handleOnConfirmAccumulator } from './utils/accumulator-helper-functions';
 import { IconAnnounce } from './announcement-components';
+import { OpenLiveChatLink } from '@deriv/components';
 
 export type TContentItem = {
     id: number;
-    text: string;
+    text: string | React.ReactElement;
```

Comment:
```suggestion
    text: React.ReactNode;
```

Can we just use `React.ReactNode`?

Reply:
We can but in this case it cos a lot of issues because in other places we use a `string`. To apply it we need to create an additional type to use where it is necessary. Are you sure we should do it?

Reply:
@maryia-matskevich-deriv I think `React.ReactNode` contains everything like elements, strings, numbers, etc. Since you are accepting elements and string then I suppose it would be ideal to just accept `React.ReactNode`, no?

Reply:
@shafin-deriv once I remove 'string' a lot of issues appear
<img width="1012" alt="Screenshot 2024-09-09 at 12 16 35" src="https://github.com/user-attachments/assets/24bea9c0-2eb8-4267-b957-060940f5a215">

Reply:
I actually wrote `React.ReactNode` but I think you are still using React.ReactElement?

Reply:
mm, I apologize, it's my fault then, I didn't notice. Thank you🙇

Reply:
✅done